scripts/check-event-schema-examples.py

```diff
@@ -42,6 +42,12 @@ def import_error(module, package, debian, error):
     import_error("jsonschema", "jsonschema", "jsonschema", e)
     raise
 
+try:
+    import referencing
```

Comment:
Since we depend on it, we should add this to `requirements.txt`, even though it will be pulled in as a transitive dependency in practice.

Reply:
Added